### PR TITLE
go-sqlcmd: Add version 1.5.0

### LINKED ISSUE #5367: [Request]: go-sqlcmd
### Prerequisites

- [X] I have searched all issues/PRs to ensure it has not already been reported or fixed.

### Criteria

- [X] Non-GUI tool
- [X] Reasonably well-known and widely used (e.g. if it's a GitHub project, it should have at least 500 stars and/or 150 forks)
- [X] English interface (or at least English documentation)
- [X] Latest stable version
- [X] Full version (i.e. not a trial version)
- [X] Fairly standard install (e.g. uses a version-specific download URL, no elaborate pre/post install scripts)

### Name

go-sqlcmd

### Description

sqlcmd is the standard Microsoft command line experience for Microsoft SQL Server and Azure SQL

### Homepage

https://github.com/microsoft/go-sqlcmd

### Download Link(s)

https://github.com/microsoft/go-sqlcmd/releases/download/v1.5.0/sqlcmd-v1.5.0-windows-x64.zip

### Some Indication of Popularity/Repute

This is the new-ish standard cross-platform command line tool to connect to SQL Server. The tool is built and published by Microsoft. 

Project has 54 forks and 268 stars which will grow as it is replacing the old sqlcmd.exe shipped with the SQL Server Management Studio Setup. 

I already prepared a manifest for this: https://github.com/FilipDeVos/Main/commit/1159632b89d27b1f3c51113db06d87a72f2c7492

### PR DESCRIPTION
**Pull request by FilipDeVos**
This PR will add the sqlcmd standalone implementation. sqlcmd is the standard Microsoft command line experience for Microsoft SQL Server and Azure SQL

Closes #5367

- [ X] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/go-sqlcmd.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+    "version": "1.5.0",
+    "description": "sqlcmd is a multi-platform command line experience for Microsoft SQL Server and Azure SQL",
+    "homepage": "https://github.com/microsoft/go-sqlcmd",
+    "license": "MIT",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/microsoft/go-sqlcmd/releases/download/v1.5.0/sqlcmd-v1.5.0-windows-x64.zip",
+            "hash": "6cd1d11a463549757efd8ef6daaa3a9413ccf85950ef9a49716041fdbdedc01d"
+        }
```
Comment (aliesbelik) [inline bucket/go-sqlcmd.json:10]:
```suggestion
        },
        "arm64": {
            "url": "https://github.com/microsoft/go-sqlcmd/releases/download/v1.5.0/sqlcmd-v1.5.0-windows-arm64.zip",
            "hash": "28ae735730f0020a0d44c9e453f56979c2da2a5e313e2f1209d3fbb121c361a1"
        }
```

File: bucket/go-sqlcmd.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    },
+    "bin": "sqlcmd.exe",
+    "checkver": "github",
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/microsoft/go-sqlcmd/releases/download/v$version/sqlcmd-v$version-windows-x64.zip"
+            }
```
Comment (aliesbelik) [inline bucket/go-sqlcmd.json:18]:
```suggestion
            },
            "arm64": {
                "url": "https://github.com/microsoft/go-sqlcmd/releases/download/v$version/sqlcmd-v$version-windows-arm64.zip"
            }
```

**Review by aliesbelik — CHANGES_REQUESTED** [review]
Guess it wouldn't be superfluous to add arm64 version as well.

File: bucket/go-sqlcmd.json (hunk 2 above)
Comment (FilipDeVos) [reply bucket/go-sqlcmd.json:25]: Added arm64 support

**Review by HUMORCE — APPROVED** [review]

**Pull request merged**